app/functions.php

```diff
@@ -154,7 +154,7 @@ function assignSpatieRole($user)
 {
     switch($user->role) {
         case 'user':
-            $user->assignRole('project-developer');
+            $user->assignRole('project_developer');
```

Comment:
Is this change intentional? This seems like it might have some pretty serious downstream consequences; changing the name of this role.

Reply:
changing it back to use dash instead of underscore